File: lever/src/main/java/com/pin/lever/GetAction.java
package com.pin.lever;

import android.graphics.Bitmap;

class GetAction extends Action<Void> {
    GetAction(Lever lever, Request data, int memoryPolicy, int networkPolicy, Object tag,
              String key) {
        super(lever, null, data, memoryPolicy, networkPolicy, 0, null, key, tag, false);
    }

    @Override void complete(Bitmap result, Lever.LoadedFrom from) {
    }

    @Override public void error(Exception e) {
    }
}

File: lever/src/main/java/com/pin/lever/GetAction.kt
package com.pin.lever

import android.graphics.Bitmap
import com.pin.lever.Lever.LoadedFrom

internal class GetAction(
    lever: Lever?,
    data: Request?,
    memoryPolicy: Int,
    networkPolicy: Int,
    tag: Any?,
    key: String?
) : Action<Void?>(
    lever,
    null,
    data,
    memoryPolicy,
    networkPolicy,
    0,
    null,
    key,
    tag,
    false
) {
    public override fun complete(result: Bitmap?, from: LoadedFrom?) {}
    public override fun error(e: Exception?) {}
}